run_tests.sh

```diff
@@ -51,23 +36,21 @@ testrepo () {
   MODPATHS=". $MODPATHS"
   for module in $MODPATHS; do
     echo "==> ${module}"
-    (cd $module && env GORACE='halt_on_error=1' CC=gcc $GO test -race \
+    (cd $module && env GORACE='halt_on_error=1' $GO test -race \
 	  ./...)
   done
 
   # check linters
-  if [[ $GOVERSION != 1.10 ]]; then
-    # linters do not work with modules yet
-    golangci-lint run --disable-all --deadline=10m \
-      --enable=gofmt \
-      --enable=vet \
-      --enable=gosimple \
-      --enable=unconvert \
-      --enable=ineffassign
-    if [ $? != 0 ]; then
-      echo 'golangci-lint has some complaints'
-      exit 1
-    fi
+  # linters do not work with modules yet
+  golangci-lint run --disable-all --deadline=10m \
+    --enable=gofmt \
+    --enable=vet \
+    --enable=gosimple \
+    --enable=unconvert \
+    --enable=ineffassign
```

Comment:
I've been using `goimports` instead of `gofmt`.

[Also using](https://github.com/decred/dcrdata/blob/71bb36a72193bb61f8bd4882d14d5d06a0aaa1ff/lint.sh#L33-L43) `staticcheck`, `misspell`, and `unparam`.  dcrseeder passes with these already, so might as well add 'em.